docs/src/design/story/STORYTANGL_WIDGET_VOCAB.md

```
@@ -764,31 +1086,33 @@ class RawCommandAccepts(BaseModel):
+by an explicit discriminator on the payload itself. The backend has the
+open-choice list and resolves the expected shape via `edge_id`. This
+keeps payloads short and matches existing webapp behavior.
 
 | `accepts.kind` | Wire payload | Notes |
 |---|---|---|
 | `pick` | `{}` (empty object) | The `edge_id` is the answer. |
 | `text` | `{ "text": str }` | |
 | `quantity` | `{ "quantity": int }` | |
-| `pieces` | `{ "piece_ids": [str, ...] }` | `min ≤ len ≤ max` |
+| `pieces` | `{ "piece_ids": [str, ...] }` | `min ≤ len ≤ max`. (Renamed from `tokens` in v1.2; the `select` rename was reverted in v1.2.1.) |
+| `place` | `{ "piece_id": str, "target_zone_ref": str | null, "edge_ref": str | null }` | Move a single piece into a single target. Exactly one of `target_zone_ref` or `edge_ref` is present. |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)